src/main/java/io/cloudsoft/spark/SparkClusterImpl.java

```diff
@@ -98,7 +98,9 @@ public void runJavaSparkPiDemo() {
             DynamicTasks.queueIfPossible(new SshEffectorTasks.SshPutEffectorTaskFactory(loc, format("%s/javaSparkPi.jar", sparkHome))
                     .contents(this.getClass().getResourceAsStream("/simple-project-1.0.jar")).newTask());
 
-            DynamicTasks.queueIfPossible(new SshEffectorTasks.SshEffectorTaskFactory(loc, format("%s/bin/spark-submit --class JavaSparkPi --master %s %s/javaSparkPi.jar", sparkHome, sparkMasterUrl, sparkHome)).newTask());
+            final String command = format("%s/bin/spark-submit --class JavaSparkPi --master %s %s/javaSparkPi.jar", sparkHome, sparkMasterUrl, sparkHome);
+            log.info("command: " + command);
+            DynamicTasks.queueIfPossible(new SshEffectorTasks.SshEffectorTaskFactory(loc, command).newTask());
```

Comment:
should just be `queue`, if we assume we're in a task queueing context (e.g. an effector call, which we are here); or if using `queueIfPossible` then it should be followed by an `orXxxx` method (else there is danger the task is just dropped)

the old code was bad so no need to fix here, but i wanted you to know...